client/verta/verta/runtime.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+import json
+from typing import Any, Dict
+
+
+class LogHandler:
+    """
+    Container class for collecting and dumping log entries.
+    """
+    def __init__(self):
+        self.attributes: Dict[str, Any] = dict()
+
+    def __repr__(self):
+        return str(self.attributes)
+
+    def add(self, **kwargs):
+        # Overwrites existing key-values
+        self.attributes.update(**kwargs)
+
+    def wipe_clean(self):
+        self.attributes = dict()
+
+# Use a single handler object that is wiped clean after each log entry
+# to minimize memory usage
+LOG_HANDLER = LogHandler()
```

Comment:
we should probably use a thread-local object vs a global? I'd check how other people have done this kind of magic association via context for inspiration

Reply:
Latest iteration is my attempt to use a thread local object for storing the context.  Admittedly, I'm new to working with the `threading` module, so this will likely need some work and I need to write a test to make sure it actually is thread safe.

Reply:
I also updated the sample script in the PR description for the latest changes.